blog/en/blog/2024/04/30/monthly-report.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Monthly Report (April 01 - April 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report is your window into the project's monthly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/XvSXCOav_monthly-report-apr-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. For detailed information, please read the monthly report.
+<!--truncate-->
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'Many hands make light work' rings true in our way, made possible by the collective dedication of our community.
+
+From 04.01 to 04.30, a total of 16 contributors made 43 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2024/04/30/txD3ooma_contributor-listi-apr.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2024/04/30/b01wMlfs_new-contributors-apr.png)
+
+## Recent Highlights Features
+
+- [Add discovery k8s dump data interface](https://github.com/apache/apisix/pull/11111)（Contributor: [hanqingwu](https://github.com/hanqingwu))
+
+- [Add max req/resp body size attributes in Kafka Logger plugin](https://github.com/apache/apisix/pull/11133)（Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
+
+- [Autogenerate Admin API key if not passed](https://github.com/apache/apisix/pull/11080)（Contributor: [Revolyssup](https://github.com/Revolyssup))
+
+## Recent Blog Recommendations
+
+- [Release Apache APISIX 3.8.1](https://apisix.apache.org/blog/2024/04/29/release-apache-apisix-3.8.1/)
+
+- [Release Apache APISIX 3.9.1](https://apisix.apache.org/blog/2024/04/29/release-apache-apisix-3.9.1/)
+
+The official website and GitHub Issues of Apache APISIX have accumulated rich documentation tutorials and usage experiences. If you encounter any issues, you can refer to the documentation, search for keywords in Issues, or participate in discussions on Issues to share your ideas and practical experiences.
```

Comment:
`accumulated rich documentation tutorials` --> `provide a wealth of documentation of tutorials, real-world use cases ...`